actionpack/lib/action_dispatch/middleware/session/cache_store.rb

```diff
@@ -9,13 +9,16 @@ module Session
     # of time.
     #
     # ==== Options
-    # * <tt>cache</tt>         - The cache to use. If it is not specified, <tt>Rails.cache</tt> will be used.
-    # * <tt>expire_after</tt>  - The length of time a session will be stored before automatically expiring.
+    # * <tt>cache</tt>            - The cache to use. If it is not specified, <tt>Rails.cache</tt> will be used.
+    # * <tt>expire_after</tt>     - The length of time a session will be stored before automatically expiring.
     #   By default, the <tt>:expires_in</tt> option of the cache is used.
+    # * <tt>cache_key_prefix</tt> - The string prepended to the session id to form the cache key.
+    #   Defaults to <tt>_session_id:</tt>
```

Comment:
Shouldn't this be a string value?
```suggestion
    #   Defaults to <tt>'_session_id:'</tt>
```